src/common/components/elements/ProfileAnalytics.tsx

```diff
@@ -0,0 +1,108 @@
+import { FC, useMemo } from 'react';
+import { IProfile } from '@/modules/feed/feed.interfaces';
+import { useAnchorWallet } from '@solana/wallet-adapter-react';
+import { Bid, useProfileAnalyticsQuery } from '../../../graphql/indexerTypes';
+import { lamportToSolIsh } from './ListingPreview';
+import { SolIcon } from './Price';
+import { LAMPORTS_PER_SOL } from '@solana/web3.js';
+
+type ProfileAnalyticsProps = {
+  profile: IProfile;
+};
+
+type SolTextHelperProps = {
+  sol?: number;
+};
+
+type AnalyticNumberProps = {
+  num?: number;
+};
+
+const SolNumber: FC<SolTextHelperProps> = ({ sol }) =>
+  sol ? (
```

Comment:
You can move this ternary inside the HTML to cut down on lines, like 
const SolNumber: FC<SolTextHelperProps> = ({ sol }) =>(
 <b className="ml-1 inline-flex items-center">
      <SolIcon className="mr-1 h-3 w-3" stroke="white" /> { sol ? sol / LAMPORTS_PER_SOL : 0}
    </b>
  )